osu.Game.Modes.Taiko/Judgements/TaikoJudgementInfo.cs

```diff
@@ -7,5 +7,76 @@ namespace osu.Game.Modes.Taiko.Judgements
 {
     public class TaikoJudgementInfo : JudgementInfo
     {
+        /// <summary>
+        /// The maximum score value.
+        /// </summary>
+        public const TaikoScoreResult MAX_SCORE = TaikoScoreResult.Great;
+
+        /// <summary>
+        /// The score value.
+        /// </summary>
+        public TaikoScoreResult Score;

```

Comment:
Weird having vairables called `Score` but enums being `ScoreResult`. I think we should call this the `Result` for now (`Score` is also used to represent a completed score of a play and also to represent the integer score displayed in the HUD).